src/partial-result-stream.ts

```diff
@@ -50,11 +50,52 @@ interface RequestFunction {
  *     that it is not ready for any more data. Increase this value if you
  *     experience 'Stream is still not ready to receive data' errors as a
  *     result of a slow writer in your receiving stream.
+ * @property {object} [columnsMetadata] An object map that can be used to pass
```

Comment:
nit: This property can only be used with Protos right ? Should we mention that in this document.

Reply:
We have mentioned an example already
`Eg: We need to pass Proto Function and Enum map to deserialize proto messages and enums, respectively`